skipchain/README.md

```diff
@@ -5,41 +5,138 @@ Skipchain
 
 # Skipchain implementation
 
+> Skipchains are authenticated data structures that combine ideas from 
+> blockchains and skiplists. Skipchains enable clients
+> to securely traverse the timeline in both forward and backward directions 
+> and to efficiently traverse short or long distances by employing 
+> multi-hop links. Backward links are cryptographic hashes of past blocks, 
+> as in regular blockchains. Forward links are cryptographic signatures of 
+> future blocks, which are added retroactively when the target block appears 
+> [1](https://www.usenix.org/system/files/conference/usenixsecurity17/sec17-nikitin.pdf).
+
 This skipchain implementation is based upon the Chainiac-paper
-(https://www.usenix.org/system/files/conference/usenixsecurity17/sec17-nikitin.pdf)
+[1](https://www.usenix.org/system/files/conference/usenixsecurity17/sec17-nikitin.pdf)
 and offers a very simple and extendable voting-based permissioned blockchain.
 Instead of having one global blockchain, the skipchain implementation allows
-users to use a personal blockchain where they can store data. In the cothority,
-the following data types are implemented:
-
-- [CISC](../cisc/README.md) - offers a key/value storage where
-a threshold of devices need to sign off on changes. Currently supported
-key/values are:
-  - SSH-keys - storing public ssh-keys that will be followed by a ssh-server for
-  passwordless logins
-  - Web-pages - store your webpage on your personal blockchain and only allow
-  changes if a threshold of administrators sign off
-  - Free values - add any key/value pair to your personal skipchain
-- [RandHound](../randhound/README.md) - produce verifiable random numbers
-and store them on a blockchain for later proofs
-- [OCS](../ocs/README.md) - a distributed, secure document sharing service that
-features access-control and auditability. The documents and access-control
-lists are stored on the skipchain.
+users to use a personal blockchain where they can store data. Skipchain is used
```

Comment:
I know this is not new, but I'm wondering what it means by "personal blockchain". Every conode stores the same chain as every other conode so the blockchain is not personal in that sense. Does "personal blockchain" have another meaning?

Reply:
@ineiti ?

Reply:
Yes, that's probably not correct anymore. That was an old idea to have everybody using his own skipchain, and then putting them together somehow. So this can probably just be removed.